src/js/player.js

```diff
@@ -1562,6 +1572,10 @@ class Player extends Component {
   techGet_(method) {
     if (this.tech_ && this.tech_.isReady_) {
 
+      if (method in middleware.allowedGetters) {
```

Comment:
Same here. I'd prefer to do the cleanup in a separate PR but might do it if it helps make the middleware part more clear as well.